Dockerfile

```diff
@@ -8,8 +8,14 @@ WORKDIR /app
 
 COPY . .
 
+RUN apt update && \
+    apt install openssh-client -y && \
+    rm -rf assets/js/* && \
```

Comment:
```suggestion
```
This line can be deleted, we are not building `libolm`.

Reply:
done